jax/_src/numpy/lax_numpy.py

```diff
@@ -2685,13 +2685,14 @@ def _pad_wrap(array: Array, pad_width: PadValue[int]) -> Array:
       _check_no_padding(pad_width[i], "wrap")
       continue
     size = array.shape[i]
-    repeats, (left_remainder, right_remainder) = np.divmod(pad_width[i], size)
-    total_repeats = repeats.sum() + 1
+    left_repeats, left_remainder = divmod(pad_width[i][0], size)
```

Comment:
I think we need to use `np.divmod` here in order for `left_remainder` to be static under JIT, otherwise the `if` statement below will lead to a concretization error. Maybe we're missing some test coverage?

Reply:
Here in normal JAX usage `pad_width` and `size` contain only constants (never tracers), and whether we use `np.divmod` or `divmod` the result will be constants. What I am doing here is enabling this code to work with symbolic dimensions, which `np.divmod` wants to cast to numerical values, while Python `divmod` is overloaded to handle symbolic dimensions.